mozilla-release/mobile/android/base/java/org/mozilla/gecko/activitystream/homepanel/stream/TopNewsRow.java

```diff
@@ -37,4 +41,10 @@ public void bind(List<TopNews> topNews) {
         adapter.swap(topNews);
     }
 
+    public void setLightTheme() {
```

Comment:
You are setting nothing, the name is misleading. A better one will be `updateTheme`.